pkg/util/modules/modules.go

```diff
@@ -23,19 +27,31 @@ type Manager struct {
 	modules map[string]*module
 }
 
+// PrivateModule is a functional option that can be used
+// when registering a module.
+func PrivateModule(m *module) {
+	m.public = false
+}
+
 // NewManager creates a new Manager
 func NewManager() *Manager {
 	return &Manager{
 		modules: make(map[string]*module),
 	}
 }
 
-// RegisterModule registers a new module with name and init function
+// RegisterModule registers a new module with name, init function, and option functions.
 // name must be unique to avoid overwriting modules
 // if initFn is nil, the module will not initialise
-func (m *Manager) RegisterModule(name string, initFn func() (services.Service, error)) {
+// if options is not given, then by default module is public
```

Comment:
Can you please reformat this entire comment to have sentences starting with capital letters, and ending with period. Last sentence can be just: "Modules are public by default".